### PR TITLE
Fixed docs

### PR DESCRIPTION
**Pull request by okolipraiz**

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+export default function App({ Component, pageProps }) {
+  return (
+      <Fragment>
+          <div className={poppins.className}>
+              <Head>
+                  <title>Quizard</title>
+                  <meta
+                      name="Students aids"
```
Comment (dpkreativ) [inline pages/_app.js:21]: Students aids?
What does this mean?

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,2 @@
+We will be using this folder to store all our app, api and secret keys.
```
Comment (dpkreativ) [inline .env.example:1]: For this one... I think we should use .env.local
Also the .env files should be listed in .gitignore

File: .env [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (dpkreativ) [inline .env:1]: Add this .env file to .gitignore too

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+.DS_Store
+*.pem
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+.pnpm-debug.log*
+
+# local env files
```
Comment (dpkreativ) [inline .gitignore:28]: Add all the .env files here

File: README.md [hunk 5]
```diff
@@ -1,4 +1,58 @@
...
+
+You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+
+## Learn More
+
+To learn more about Next.js, take a look at the following resources:
+
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+
+## Project Scope
```
Comment (dpkreativ) [inline README.md:51]: I think this "Project Scope" and its content should come before "Getting Started"

File: pages/_app.js (hunk 1 above)
Comment (dpkreativ) [inline pages/_app.js:21]: This sounds off... please explain why we need it, and why it's called "Students aids"

File: pages/index.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { Fragment } from 'react';
+
+
+export default function Home() {
+  return (
+      <Fragment>
```
Comment (dpkreativ) [inline pages/index.js:6]: Is there a difference between using "Fragment" and just using <>

**Review by dpkreativ — CHANGES_REQUESTED** [review]
Hi Praiz. Nice work. Please review the comments I've left on the codebase and let me know how it goes.
Thanks and have a wonderful day!

File: .env.example (hunk 2 above)
Comment (okolipraiz) [reply .env.example:1]: I have changed it to env.local, seems like they stopped allowing the usage of only .env

File: pages/_app.js (hunk 1 above)
Comment (okolipraiz) [reply pages/_app.js:21]: These are just for the start they can and will be changed later when the SEO are ready.

File: README.md (hunk 5 above)
Comment (okolipraiz) [reply README.md:51]: I dont agree with you, firstly this is the frontend of the app, not the whole product, in the sense that anyone that has access to the repo is a FE person that wants to quick start with the project. I dont know if you understand me shaa

File: pages/index.js (hunk 6 above)
Comment (okolipraiz) [reply pages/index.js:6]: They are the same thing sir, you can check the react docs for more confirmation, <Fragment> Is more descriptive. that's why I am using it unlike <> which both also returns an extra div

Comment (dpkreativ) [reply pages/index.js:6]: Okay thanks for clearing that up

File: README.md (hunk 5 above)
Comment (dpkreativ) [reply README.md:51]: I understand. It's cool.

**Review by dpkreativ — APPROVED** [review]
Good to go!

**Pull request merged**